src/domain/cart/CartPage.tsx

```diff
@@ -1,5 +1,52 @@
-export const CartPage = (): JSX.Element => {
+import './style.css'
+import { useAppDispatch } from "../../store/store";
+import { removeFromCart } from "./cartSlice";
+import { CartItem, CartState } from "./types"
+
+type CartProps = CartState
+
+
+
+export const CartPage = ({cartItems}:CartProps): JSX.Element => {
+  const dispatch = useAppDispatch();
```

Comment:
This is a top level component so use the useSelector hook to get the state instead of passing it from the props.